### PR TITLE
Dev/changes

### PR DESCRIPTION
**Pull request by Joelzac537**
**Introduction**
Reloading of data in Dashboard

**Description**
Made changes so that when data is added dashboard reloads immediately

**How to Test**
After submitting data, dashboard will automatically reload the number of data according to category

### CODE REVIEW

File: public/scripts/worktrollyUI.js [hunk 1]
```diff
@@ -164,11 +164,8 @@ $("#submitCreateNewTask").click(function() {
     var createNewTaskFunction = firebase.functions().httpsCallable('createNewTask');
     createNewTaskFunction({ Title: title, Description: des, Priority: priority, Difficulty: difficulty, Creator: creator, Assignee: assignee, EstimatedTime: estimatedTime, Status: status, Category: category, SprintNumber: sprintNumber, StoryPointNumber: storyPointNumber }).then(result => {
         console.log(result.data);
-        newPage = "dashboard";
-        uiLoader();
+        location.reload();
     });
-    $('input').val('');
-    $('select').val('');
 });
```
Comment (shivamgoyal7) [inline public/scripts/worktrollyUI.js:169]: undo this change.
And add .onSnapshot instead of .get()

`use this format to read`
db.collection("cities").doc("SF")
    .onSnapshot(function(doc) {
     ........
    });

do these changes everywhere in dashboard.js

Comment (shivamgoyal7) [reply public/scripts/worktrollyUI.js:169]: and also remove the catch method after reading the data.

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
do the changes in reading the data in the dashboard.js and test it properly

File: public/scripts/worktrollyUI.js (hunk 1 above)
Comment (Joelzac537) [reply public/scripts/worktrollyUI.js]: Done the appropriate changes

**Review by shivamgoyal7 — APPROVED** [review]
ready to merge

File: public/scripts/dashboard.js [hunk 2]
```diff
@@ -1,14 +1,11 @@
 function getDashboardData() {
-    db.collection("Main").doc("RawData").get()
-        .then(function(doc) {
+    db.collection("Main").doc("RawData")
+        .onSnapshot(function(doc) {
             currentSprint = doc.data().CurrentSprintId;
             console.log(currentSprint);
             selectedSprint = currentSprint;
             setDataIntoDashboard();
         })
```
Comment (shivamgoyal7) [inline public/scripts/dashboard.js:8]: Add semicolon

File: public/scripts/dashboard.js [hunk 3]
```diff
@@ -64,27 +61,23 @@ function getNewSprintId() {
...
+                var totalMarketingTask = doc.data().TotalMarketingTask;
+                document.getElementById("totalDevelopmentTaskNewSprint").innerHTML = totalDevelopmentTask;
+                document.getElementById("totalBusinessTaskNewSprint").innerHTML = totalBusinessTask;
+                document.getElementById("totalMarketingTaskNewSprint").innerHTML = totalMarketingTask;
+            } else {
                 document.getElementById("totalDevelopmentTaskNewSprint").innerHTML = 0;
                 document.getElementById("totalBusinessTaskNewSprint").innerHTML = 0;
                 document.getElementById("totalMarketingTaskNewSprint").innerHTML = 0;
             }
         })
```
Comment (shivamgoyal7) [inline public/scripts/dashboard.js:82]: add semicolon

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
do this

File: public/scripts/dashboard.js (hunk 3 above)
Comment (Joelzac537) [reply public/scripts/dashboard.js]: added

File: public/scripts/dashboard.js (hunk 2 above)
Comment (Joelzac537) [reply public/scripts/dashboard.js]: added

**Review by shivamgoyal7 — APPROVED** [review]
able to merge

**Review by viv9k — APPROVED** [review]
ship it
